CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+**0.3.4**
+1. Added possibility to perform backup and restore non-coin boxes to bootstrap a new sidechain with a history of non-coin data taken from other sidechain.
```

Comment:
I would change this in something like "Added the possibility to perform a backup of a sidehcain non coin-boxes and restore these boxes into a new bootstrapped sidechain (with the same type).

Reply:
good